sass_processor/management/commands/compilescss.py

```diff
@@ -2,31 +2,32 @@
 from __future__ import unicode_literals
 
 import os
+
```

Comment:
I apply [isort](https://github.com/timothycrosley/isort) to all of my files, maybe we should start doing this everywhere and add a check for it in Travis along with flake8?